sdk/identity/Azure.Identity/src/CredentialPipeline.cs

```diff
@@ -39,7 +39,7 @@ public IConfidentialClientApplication CreateMsalConfidentialClient(string tenant
             return ConfidentialClientApplicationBuilder.Create(clientId).WithHttpClientFactory(new HttpPipelineClientFactory(HttpPipeline)).WithTenantId(tenantId).WithClientSecret(clientSecret).Build();
         }
 
-        public MsalPublicClient CreateMsalPublicClient(string clientId, string tenantId = default, string redirectUrl = default, bool attachSharedCache = false)
+        public MsalPublicClient CreateMsalPublicClient(string clientId, string tenantId = default, string redirectUrl = default, bool attachSharedCache = true)
```

Comment:
Looks like `attachSharedCache` defaults `false` everywhere else, why is it `true` here?